TDSQAService/OSS.TIS/SQL/TDSItemBank/7_create_procedure_spDeleteAssessment.sql

```diff
@@ -13,7 +13,7 @@ BEGIN
 	/***********************************************************************************************************************
 	  File: 7_create_procedure_spDeleteAssessment.sql
 
-	  Desc: Delete an assessment from the TIS databases.
+	  Desc: Delete all assessments from the TIS databases for a given test package key.

```

Comment:
Update comment for accuracy.  Honestly, this stored procedure should probably be renamed `spDeleteTestPackage`.